source/guides/guides/stubs-spies-and-clocks.md

```diff
@@ -197,7 +197,7 @@ When you use aliasing with the {% url `.as()` as %} command, we also correlate t
 
 When stubs are created by calling the method `.withArgs(...)` we also visually link these together.
 
-When you click on a stub or spy, we also output **incredibly** helpful debugging information.
+When you click on a stub or spy, we also output **remarkably** helpful debugging information.
```

Comment:
This one's tricky. In hindsight I think both words are just as good in this context. Just marking nuances.